packages/container/lib/registry.js

```diff
@@ -546,7 +554,14 @@ export default class Registry {
     return assign({}, fallbackKnown, localKnown, resolverKnown);
   }
 
-  isValidFullName(fullName) {
+  isValidFullName(fullName, options) {
+    if(EMBER_MODULE_UNIFICATION) {
+      /* With a rawString, the fullName doesn't need to contain a : */
+      if (options && options[RAW_STRING_OPTION_KEY] && fullName) {
+        return true;
```

Comment:
This needs review with @rwjblue @dgeb I'm not quite sure what to do here. The gist is that previously we required `type:name` but this check happens at low levels and now `type` alone is permitted (the name ends up in the rawString). Perhaps we should move this check out of the deep parts of the container instead of removing it in this PR.

Reply:
@mixonic I believe type alone is no longer permitted so we can remove this.